### ISSUE TITLE
Does not preprocess all the documents.

### ISSUE THREAD

**Issue by SAIKIRAN0802** [deepset-ai/FARM#597]
**Question**
Put your question here
Does not preprocess all the documents.during the finetuning process for annotations exported through haystack annotation-tool.take a look at the output below it only processes 5/13

**Additional context**
Add any other context or screenshots about the question (optional).
Preprocessing Dataset demo.json:  38%|███▊      | 5/13 [01:04<01:42, 12.81s/ Dicts]
10/21/2020 09:37:49 - WARNING - farm.modeling.prediction_head -   Some unused parameters are passed to the QuestionAnsweringHead. Might not be a problem. Params: {"training": false, "num_labels": 2, "ph_output_type": "per_token_squad", "model_type": "span_classification", "config": {"training": true, "layer_dims": [768, 2], "num_labels": 2, "ph_output_type": "per_token_squad", "model_type": "span_classification", "task_name": "question_answering", "no_ans_boost": 0.0, "context_window_size": 100, "n_best": 5, "n_best_per_sample": 1, "name": "QuestionAnsweringHead"}, "label_tensor_name": "question_answering_label_ids", "label_list": ["start_token", "end_token"], "metric": "squad", "name": "QuestionAnsweringHead"}
10/21/2020 09:37:49 - INFO - farm.modeling.optimization -   Loading optimizer `TransformersAdamW`: '{'correct_bias': False, 'weight_decay': 0.01, 'lr': 1e-05}'
10/21/2020 09:37:49 - INFO - farm.modeling.optimization -   Using scheduler 'get_linear_schedule_with_warmup'
10/21/2020 09:37:49 - INFO - farm.modeling.optimization -   Loading schedule `get_linear_schedule_with_warmup`: '{'num_training_steps': 1897, 'num_warmup_steps': 379}'
Train epoch 0/0 (Cur. train loss: 0.0002): 100%|██████████| 1897/1897 [16:18<00:00,  1.94it/s]
10/21/2020 09:54:08 - INFO - haystack.reader.farm -   Saving reader model to drive/My Drive/model5

**Comment by SAIKIRAN0802** [comment]
Seems to be fixed for me for now
